x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/action/TransportDeleteExpiredDataActionTests.java

```diff
@@ -93,7 +97,7 @@ public void testDeleteExpiredDataIterationWithTimeout() {
 
         Supplier<Boolean> isTimedOutSupplier = () -> (removersRemaining.getAndDecrement() <= 0);
 
-        transportDeleteExpiredDataAction.deleteExpiredData(removers.iterator(), finalListener, isTimedOutSupplier, true);
+        transportDeleteExpiredDataAction.deleteExpiredData(removers.iterator(), .0f, finalListener, isTimedOutSupplier, true);
```

Comment:
It seems dodgy to test two error conditions together: a timeout and requests per second = 0.  Maybe the `.0f` was a typo?  I think this test should just test the timeout.